LanguageFeatures/Parts-with-imports/scope_lib3.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Common library for `scope*` tests. Exports `scope_lib2.dart`.
+///
+/// @description Common library for `scope*` tests. Exports `scope_lib2.dart`.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=enhanced-parts
+
+import 'scope_lib2.dart';
```

Comment:
It is used in `scope_A02_t06_part1.dart`.
```dart
import 'scope_lib3.dart';
testPart1() {
  // From scope_lib2.dart
  Expect.equals("scope_lib2", libId); // Check that 'scope_lib2.dart' is exported
...
}
```

Reply:
I can't see how that would make a difference: There is nothing in this library that uses any declaration exported by 'scope_lib2.dart', and the import in this library makes no difference for the set of declarations exported by this library (so it can't make a difference for 'scope_A02_t06_part1.dart'). We do need the export directive below, but why would we have the import in this library?

Reply:
Got it, thanks!